### PR TITLE
IBX-3796: Document segment group protection

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | [IBX-3796](https://issues.ibexa.co/browse/IBX-3796) |
| Versions      | 4.x |
| Edition       | all |

Document config needed to protect segment groups

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

**Review by webhdx — APPROVED** [review]
One important remark - Segment Groups are only protected in the UI. You are still allowed to modify them using APIs.

File: docs/release_notes/ibexa_dxp_v4.2.md [hunk 1]
```diff
@@ -253,6 +253,16 @@ Thanks to splitting SQL upgrade files into separate product editions, the produc
...
 Now, Events are sent while performing operations within Taxonomy, which considerably extends the Taxonomy feature.
 
-### Protected Segment Groups
+### Protected segment groups
 
-You can now set existing Segment Groups as protected and make them unable to be modified through the user interface.
+You can now set existing [segment groups](https://doc.ibexa.co/en/latest/administration/admin_panel/segments_admin_panel/) as protected and prevent them from being modified through the user interface.
+It's intended to stop users from breaking data integrity of segments/segment groups maintained by other features or external system integrations, such as [Customer Portal](https://doc.ibexa.co/en/latest/customer_management/customer_portal/) and [CDP](https://doc.ibexa.co/en/latest/cdp/cdp/).
+
+Do do it, in your configuration, add the following key for each segment group that you intend to protect:
```
Comment (julitafalcondusza) [inline docs/release_notes/ibexa_dxp_v4.2.md:261]:
```suggestion
To do it, in your configuration, add the following key for each segment group that you intend to protect:
```

**Review by julitafalcondusza — CHANGES_REQUESTED** [review]
See my comment.

**Comment by MagdalenaZuba** [comment]
Remember about adding it to 3.3 and 2.5.

File: docs/release_notes/ibexa_dxp_v4.2.md [hunk 2]
```diff
@@ -253,6 +253,16 @@ Thanks to splitting SQL upgrade files into separate product editions, the produc
 
 Now, Events are sent while performing operations within Taxonomy, which considerably extends the Taxonomy feature.
 
-### Protected Segment Groups
+### Protected segment groups
 
-You can now set existing Segment Groups as protected and make them unable to be modified through the user interface.
+You can now set existing [segment groups](https://doc.ibexa.co/en/latest/administration/admin_panel/segments_admin_panel/) as protected and prevent them from being modified through the user interface.
```
Comment (dabrt) [inline docs/release_notes/ibexa_dxp_v4.2.md:258]:
```suggestion
You can now set existing [segment groups](https://doc.ibexa.co/en/latest/administration/admin_panel/segments_admin_panel/) as protected, and prevent them from being modified through the user interface.
```

**Review by julitafalcondusza — APPROVED** [review]

**Review by MagdalenaZuba — APPROVED** [review]

**Pull request merged**